### ISSUE TITLE
[WISE Website Feedback]: Text size variation looks terrible

### ISSUE THREAD

**Issue by spydmobile** [WISE-Developers/Project_issues#169]
### Your thoughts?

The difference ins size between elements is off-putting. The sizes should be properly scaled as per good UI/UX:
![Screenshot 2023-01-19 at 4 15 12 PM](https://user-images.githubusercontent.com/1065978/213583323-0bb1db46-4614-465a-8c09-793845b86ab3.png)

**Comment by BadgerOnABike** [comment]
Agreed, the size of discord vs github is weird. and the break from seemingly H1 to p is really stark. Perhaps H2 or 3 could be more appropriate. Some strange justification going on too.

**Comment by spydmobile** [comment]
> Agreed, the size of discord vs github is weird. and the break from seemingly H1 to p is really stark. Perhaps H2 or 3 could be more appropriate. Some strange justification going on too.

I may very well resort to using some of those ass kicking svg badges ;-) See:
https://github.com/WISE-Developers

**Comment by BadgerOnABike** [comment]
Oh sweet, thats really nice! I think gh up and discord down a bit, meet in the middle!
